### ISSUE TITLE
 Unexpected token (57:6)

### ISSUE THREAD

**Issue by tonylinyy** [react-component/input-number#1]
here is my step:
```
1.var InputNumber = require('rc-input-number');
2.<InputNumber min='1' max='100' value='10' disabled={false}/>
3.I use watcify:watchify -t [reactify --es6] ./client/web/index.js -o ./public/build/bundle.js
```
always show the error:
Error: Parsing file /node_modules/rc-input-number/lib/InputNumber.js: Unexpected token (57:6)

**Comment by yiminghe** [comment]
fix at 1.0.1

**Comment by tonylinyy** [comment]
已经可以用了,谢谢
